.github/workflows/generate_chapters.yml

```
@@ -75,19 +75,20 @@ jobs:
         echo "regenerating chapters"
         npm run generate
     - name: Create Pull Request
-      uses: peter-evans/create-pull-request@v2
+      id: cpr
+      uses: peter-evans/create-pull-request@v3
       with:
         title: Update datestamps and generate chapters
         branch-suffix: timestamp
         commit-message: Generate Chapters
         body: |
-            Updated datestamps and regenerated the chapters
-            - Please merge today or we'll get new datestamps tomorrow and you'll have to merge again!
-            - Auto-generated by [create-pull-request][1] GitHub Action
+          Updated datestamps and regenerated the chapters
+          - Please merge today or we'll get new datestamps tomorrow and you'll have to merge again!
+          - Auto-generated by [create-pull-request][1] GitHub Action
 
-            [1]: https://github.com/peter-evans/create-pull-request
-        labels: generate chapters
+          [1]: https://github.com/peter-evans/create-pull-request
+        labels: |
+          generate chapters
```

I do not see the pressing need to make this change, but it does not make a difference as such. We usually use this style for multi-line values.

```suggestion
        labels: generate chapters
```